### ISSUE TITLE
Subscription returns `undefined` with superjson

### ISSUE THREAD

**Issue by mmurto** [jsonnull/electron-trpc#123]
If is use `superjson` as transformer, this procedure:

```ts
open: publicProcedure.subscription(async () => {
  return observable<{ open: boolean }>((emit) => {
    const setFileopenVisible = (data: boolean) => {
      console.log("setFileopenVisible", data);
        emit.next({ open: data });
    };
    ee.on("setFileopenVisible", setFileopenVisible);
    return () => {
      ee.off("setFileopenVisible", setFileopenVisible);
    };
  })
}),
```

returns undefined when called here:

```ts
trpc.ui.openFileVisible.open.useSubscription(undefined, {
  onData: (data) => {
    console.log("data", data);
  },
});
```

The data is properly returned, if I remove the transformer from the api and the client.

Using electron-trpc version 0.4.2.

**Comment by spencekim** [comment]
I am seeing this as well. Did you find a workaround?

**Comment by jsonnull** [comment]
A fix for this is out in `electron-trpc@0.4.5`, which includes updated dependencies.

Let me know if you run into any more issues!
